frontend/app/src/pages/superadmin/Header/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import {
+  NavWrapper,
+  AlternateWrapper,
+  ButtonWrapper,
+  Title,
+  Button,
+  AlternateTitle,
+  ExportButton,
+  ExportText,
+  Month,
+  ArrowButton,
+  DropDown,
+  DropDownOption,
+  Flex,
+} from "./headerstyles"
+import signout from "./icons/signout.svg";
+import arrowback from "./icons/arrowback.svg";
+import arrowforward from "./icons/arrowforward.svg";
+
+
+export const Header = () => (
+    <>
+  
+          <NavWrapper>
+            <Title>
+              Bounties
+              <AlternateTitle>Super Admin</AlternateTitle>
+            </Title>
+            <Button>
+              Sign out 
+              <img src={signout} alt="Sign Out" />
+
+            </Button>
+          </NavWrapper>
+          <AlternateWrapper>
+            <Flex>
+              <ButtonWrapper>
+                <ArrowButton>
+                  <img src={arrowback} alt=""  />
+                </ArrowButton>
+                <ArrowButton><img src={arrowforward} alt=""  /></ArrowButton>
+              </ButtonWrapper>
+              <Month>01 Oct-31 Dec 2023</Month>
```

Comment:
lets not hardcode this month, lets use moment to figure out the current date and then set these dynamically